.env.example

```diff
@@ -45,8 +45,8 @@ USE_OPENAI_EMBEDDING=  #TRUE
 #leave blank to use defaults hermes 70b for small tasks & 405b for medium/large tasks
 OPENROUTER_MODEL=
 SMALL_OPENROUTER_MODEL=
-MEDIUM_OLLAMA_MODEL=
-LARGE_OLLAMA_MODEL=
+MEDIUM_OPENROUTER_MODEL=
+LARGE_OPENROUTER_MODEL=
```

Comment:
is this going to break backwards with this evn change?

Reply:
These were typo corrections where OpenRouter models were defined. The previous two values were duplicates of OLLAMA model vars elsewhere in the file.